src/components/FormSections/Confirmation.jsx

```diff
@@ -167,28 +153,13 @@ function Confirmation(props){
 
         <div className="usa-alert usa-alert--info">
             <div className="usa-alert__body">
+            <h4>{content.acknowledge_heading}</h4>
                 <p>{content.acknowledge_text}</p>
             </div>
         </div>
 
-        <div className={error ? 'error-container' : ''}>
-            <Checkbox
-                id="acknowledge-check"
-                name="acknowledge-check"
-                required
-                defaultChecked={hasAcknowledged}
-                label="I can confirm my information is correct to the best of my knowledge."
-                onChange={(e) => acknowledgeCheckbox(e.target.checked)}
-                />
-            {error &&
-                <span id="first-name-error" role="alert" className='error-text'>
-                    Checkbox must be checked to continue.
-                </span>
-            }
-        </div>
-
-            <Button onClick={() => {checkboxValid(), hasAcknowledged && GenerateFilledPDF(props.fieldData)}} type="submit">
-                Confirm and Download Form
+            <Button type="submit">
+                Confirm and continue
             </Button>
```

Comment:
We need the checkbox function reverted. In discussion with Craig and Susan this was a error in the design and it should remain.